yaq/src/leaderboard.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+    <link rel="stylesheet" href="public/styles/leaderboard.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="header">
+        <span class="back-arrow">←</span>
```

Comment:
Use a `<svg>` instead of a literal arrow. You can copy the arrow SVG from Figma.
Also, since users will click on the back arrow, it would make sense to put it in a `<button>` tag for easy _accessibility_.